matchms/filtering/require_minimum_of_high_peaks.py

```diff
@@ -0,0 +1,34 @@
+from ..typing import SpectrumType
+from .select_by_relative_intensity import select_by_relative_intensity
+
+
+def require_minimum_of_high_peaks(spectrum_in: SpectrumType, no_peaks: int = 5,
+                                  intensity_percent: float = 2.0) -> SpectrumType:
+
+    """Returns None if the number of peaks with relative intensity
+       above or equal to intensity_percent is less than no_peaks.
+
+    Parameters
+    ----------
+    spectrum
```

Comment:
Match docstring with function spec
```suggestion
    spectrum_in
```